lib/src/bool/has_lowercase_chars_validator.dart

```diff
@@ -24,21 +24,23 @@ class HasLowercaseCharsValidator extends BaseValidator<String> {
       FormBuilderLocalizations.current.containsLowercaseCharErrorText(atLeast);
 
   /// {@template lower_case_template}
-  /// This regex matches any character that is not a lowercase letter (a-z) or ñ.
+  /// This regex matches any character that is not a lowercase letter (a-z).
   ///
   /// - It includes special characters, digits, and uppercase letters.
   /// - It can be used to find non-lowercase characters.
   ///
   /// Examples: A, 1, @
   /// {@endtemplate}
-  static final RegExp _lowerCase = RegExp('[^a-zñ]');
+  static final RegExp _lowerCase = RegExp('[a-z]');
```

Comment:
Why remove ñ?
On spanish language, is a common letter

Reply:
It makes no sense right now. There are much more languages with strange letters. Do you know a regex that contains all those as well? If so we can put it in here off course.